src/Orckestra.Composer.Website/UI.Package/Typescript/Composer.SingleCheckout/ReviewCartSingleCheckoutController.ts

```diff
@@ -80,6 +72,46 @@ module Orckestra.Composer {
                             });
 
                         this.Cart.LineItemDetailViewModels.splice(index, 1);
+                    },
+                    changeRecurringMode(e, item) {
```

Comment:
Can we have sepearte TS controller for all these, for example name it ChangeRecurringFrequencyCheckoutControleer